pkg/client/client.go

```diff
@@ -685,6 +725,26 @@ func (o *OptimizelyClient) RemoveOnTrack(id int) error {
 	return nil
 }
 
+func (o *OptimizelyClient) getTypedValue(value string, variable entities.Variable) (out interface{}, err error) {
+	out = value
+	switch varType := variable.Type; varType {
```

Comment:
Why do we need to use a new variable `varType`? Does this not work?
```suggestion
	switch variable.Type {
```